### ISSUE TITLE
Not movable pieces

### ISSUE THREAD

**Issue by hungdnguyen** [JolyRoger/mediocrechess#1]
Hi, I tried to run it on both of Windows and Mac, but I could not move white pieces although when I click at them, I can see the yellow square but onMouseDrag or something like that does not work.

**Comment by JolyRoger** [comment]
Thank you for your report. 
This is custom chessboard, which supports only move with two clicks. First — onto the square with piece, and second — the square what you woud like to move to. 
If it doesn't work also... 
Well. Sorry for inconvenience. I wrote it for myself  and didn't touch this project for long time.
The I will try to look at the project closer and fix the problem.

**Comment by hungdnguyen** [comment]
Yeah, the two clicks do not work though. But well I don't want to waste your time. I found another engine which runs Mediocre Chess as well. I just want to learn chess algorithm instead of being involved with generating moves and GUI stuff. Hopefully, I could learn whole Mediocre Chess and move on.

Thank you so much,
Hung Nguyen

**Comment by JolyRoger** [comment]
Yes, Mediocre is just an engine, and could be plugged into any appropriate UI.
Good luck to you in your challenge.
